sync_diff_inspector/diff.go

```diff
@@ -152,23 +205,77 @@ func (df *Diff) Equal(ctx context.Context) error {
 			// finish read the tables
 			break
 		}
-		log.Debug("generate chunk", zap.Int("chunk id", c.ChunkRange.ID))
-
+		tableDiff := df.downstream.GetTables()[c.TableIndex]
+		log.Info("generate chunk", zap.Int("chunk id", c.ChunkRange.ID), zap.String("table", dbutil.TableName(tableDiff.Schema, tableDiff.Table)))
 		pool.Apply(func() {
-			res, err := df.consume(ctx, c)
-			if err != nil {
-				// TODO: catch error
-			}
-			// TODO: handle res
-			if res {
-
+			tableDiff := df.workSource.GetTables()[c.TableIndex]
```

Comment:
this function needs a lock.
I suggest save the `schema` `table` into Chunk

Reply:
I cannot see there is any concurrent writes here

Reply:
ok, There is no race.

but In this code we will repeated compare the same table `chunksCount` times. we need reduce the compare times. 🤔